src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newObj = { ...state };
+  const array = [];
+
```

Comment:
What does array mean?
Better to use names, what others devs can understand
Maybe actionHistory

Reply:
agree, fix this variable's name